ProcessMaker/processmaker
# FOUR-11405 | When Closing a Created Asset (From ‘+Asset’), After Publishing and Closing It, Redirect Back to the Project

# Issue
Ticket: [FOUR-11405](https://processmaker.atlassian.net/browse/FOUR-11405)

In a Project's Asset Listing, when a new asset is created, after that asset has been saved, the user should be redirected back to the Project they created the asset from.

# Solution
- Implement redirects for Processes, Screens, Scripts, Data Sources, and Decision Tables from the respective creation modals.

# How to Test
1. Go to branch `observation/FOUR-11405` in `processmaker`.
2. Go to branch `observation/FOUR-11405` in `package-projects`.
3. Go to branch `observation/FOUR-11405` in `package-data-sources`.
4. Go to branch `observation/FOUR-11405` in `package-decision-engine`.
5. Go to branch `observation/FOUR-11405` in `package-ai`.
6. Go to Designer → Projects. Open a Project.
7. Create a new Process from within the Asset Listing.
	- You should be redirected to the Process Modeler.
8. Save your Process.
	- You should be redirected back to your Project.
9. Repeat for all asset types.
10. Ensure that when creating an asset (a Process, Screen, or Script) from a Process using AI Generation, you are redirected back to the Process Modeler.

ci:next
ci:package-projects:observation/FOUR-11405
ci:package-data-sources:observation/FOUR-11405
ci:package-decision-engine:observation/FOUR-11405
ci:package-ai:observation/FOUR-11405
ci:deploy

# Code Review Checklist


[FOUR-11405]: https://processmaker.atlassian.net/browse/FOUR-11405?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
